dapps/v2Explorer/android/app/src/main/java/com/walletconnect/web3modal/rnsample/MainApplication.java

```diff
@@ -24,7 +24,7 @@ protected List<ReactPackage> getPackages() {
           @SuppressWarnings("UnnecessaryLocalVariable")
           List<ReactPackage> packages = new PackageList(this).getPackages();
           // Packages that cannot be autolinked yet can be added manually here, for example:
-           packages.add(new CustomPackage());
+          // packages.add(new CustomPackage());
```

Comment:
Remove?

Reply:
it's on purpose, is an example for the line above:
`// Packages that cannot be autolinked yet can be added manually here, for example:`